tools/basic-illumination/macros.xml

```diff
@@ -15,8 +15,8 @@
         </citations>
     </xml>
 
-    <token name="@TOOL_VERSION@">1.0.3</token>
-    <token name="@VERSION_SUFFIX@">1</token>
+    <token name="@TOOL_VERSION@">1.1.1</token>
+    <token name="@VERSION_SUFFIX@">2</token>
```

Comment:
@CameronFRWatson this should be reset if we bump the main version. It is not super important but maybe next time.